src/main.py

```diff
@@ -1,22 +1,17 @@
 import utils
 import matplotlib.pyplot as plt
+from sklearn.model_selection import train_test_split
 
 """Task 0 - Split training and evaluation data"""
-
 # Read data
-all_docs, all_labels = utils.read_documents('input/all_sentiment_shuffled.txt')
-# Split point between training and evaluation
-split_point = int(0.80*len(all_docs))
+X, y = utils.read_documents('input/all_sentiment_shuffled.txt')  
 
-# Get training and evaluation data
-train_docs = all_docs[:split_point]
-train_labels = all_labels[:split_point]
-eval_docs = all_docs[split_point:]
-eval_labels = all_labels[split_point:]
+# Split point between training and evaluation
+X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.2)
```

Comment:
maybe add `shuffle=False` to train_test_split()
i'm thinking since in Task 3 we need to print the row nb, maybe we need to keep the order as is

Reply:
We can get the row nb from `train_test_split` if we provide another array after X and y.
We get the lists with indices for the train and test sets.
```python
X_train, X_test, y_train, y_test, indices_train, indices_test = train_test_split(all_docs, all_labels, np.arange(len(all_docs)), test_size=0.2)

Reply:
I wasn't aware of that, cool thanks

Reply:
wait, okay yea i see now lol, that makes sense